onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -815,6 +823,10 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
               "[ERROR] [OpenVINO] The value for the key 'export_ep_ctx_blob' "
               "should be a boolean i.e. true or false. Default value is false.\n");
         }
+      } else if (key == "use_device_mem") {
```

Comment:
as implemented i think we'd get runtime errors if OVEP was built with OV version < 2024.4